.github/workflows/inappmessaging.yml

```diff
@@ -41,32 +41,32 @@ jobs:
     - name: FirebaseInAppMessaging
       run: scripts/third_party/travis/retry.sh scripts/pod_lib_lint.rb ${{ matrix.podspec}}
 
-  tests:
-    # Don't run on private repo unless it is a PR.
-    if: (github.repository == 'Firebase/firebase-ios-sdk' && github.event_name == 'schedule') || github.event_name == 'pull_request'
+#   tests:
+#     # Don't run on private repo unless it is a PR.
+#     if: (github.repository == 'Firebase/firebase-ios-sdk' && github.event_name == 'schedule') || github.event_name == 'pull_request'
 
-# TODO(#12770): Update to macos-14 when tests are updated for Xcode 15.
```

Comment:
How about keeping the GitHub Action for a build test and changing build.sh to build instead of test?